server/lambda/etsToSbc/sbc-phac.js

```diff
@@ -0,0 +1,53 @@
+const dayjs = require('dayjs');
```

Comment:
Re. the name of this file, it looks like it just sends ETS not PHAC?

Reply:
nice catch!